packages/extension-base/src/services/chain-service/index.ts

```diff
@@ -816,8 +816,39 @@ export class ChainService {
       return;
     }
 
-    const onUpdateStatus = (status: _ChainConnectionStatus) => {
+    const onUpdateStatus = async (status: _ChainConnectionStatus) => {
       const slug = chainInfo.slug;
+      const isActive = this.getChainStateByKey(slug).active;
+      const isConnectProblem = status !== _ChainConnectionStatus.CONNECTING && status !== _ChainConnectionStatus.CONNECTED;
+      const isLightRpc = endpoint.startsWith('light');
+
+      if (isActive && isConnectProblem && !isLightRpc) {
+        const reportApiUrl = 'https://api-cache.subwallet.app/api/health-check/report-rpc';
+        const requestBody = {
+          chainSLug: slug,
```

Comment:
typo in chainSlug

Reply:
this typo is from API, I also feedback to BE team